faq/README.md

```diff
@@ -157,4 +157,59 @@ and we've had many reports where the problem turned out to be in user code.
 
 See [Blobs.java] in the `driver-examples` module for some examples and explanations.
 
-[Blobs.java]: https://github.com/datastax/java-driver/tree/3.0.x/driver-examples/src/main/java/com/datastax/driver/examples/datatypes/Blobs.java
\ No newline at end of file
+
+### Why am I seeing messages about tombstone_warn_threshold or tombstone_fail_threshold being exceeded?
```

Comment:
Maybe you should specify that such error messages do not appear in the driver logs, but in Cassandra logs instead.